chain-tx-enclave-next/enclave-ra/ra-enclave/src/certificate.rs

```diff
@@ -16,7 +16,8 @@ impl Certificate {
     /// Checks if current certificate is valid or not
     pub fn is_valid(&self, validity_duration: Duration) -> bool {
         let current_time = Utc::now();
-        self.created + validity_duration >= current_time
+        // add extra 15 seconds here to avoid the impact of network transfer delay
+        current_time - self.created <= validity_duration - Duration::seconds(15)
```

Comment:
can this underflow?

Reply:
possible, how about adding underflow check above